src/repositories/location.repository.ts

```diff
@@ -0,0 +1,33 @@
+import { Repository } from '../core/repository';
+import {
+  LocationRepositorySearchResponseRootObject,
+  LocationRepositoryInfoResponseRootObject,
+} from '../responses';
+
+export class LocationRepository extends Repository {
+  public async search(latitude: number, longitude: number, searchQuery?: string): Promise<LocationRepositorySearchResponseRootObject> {
```

Comment:
Let's move this method to `LocationSearch.index()` or `LocationSearch.main()`?
And it would be nice to add convenient method here https://github.com/dilame/instagram-private-api/blob/master/src/services/search.service.ts

Reply:
Should the LocationSearch be located in the /repositories or /service (and which filename would it have)?

Reply:
This is surely repository. 
`location-search.repository.ts`

Reply:
Should be done